Users/models/categoryDao.js

```diff
@@ -0,0 +1,23 @@
+import prisma from '../../prisma/index';
+
+const findCategories = async () => {
+  return await prisma.$queryRaw(`
+  SELECT
+    categories.id, categories.name
+  FROM 
+    categories
+  `);
+};
+
```

Comment:
```suggestion
  return await prisma.$queryRaw(`
    SELECT
      categories.id, categories.name
    FROM 
      categories
  `);
};

```
return 보다 안쪽으로 코드 넣어주시는 것이 좋을 것 같습니다 :) 아래 다른 코드들도 수정해주세요~

Reply:
소헌님! return보다 안쪽으로 코드를 넣어준다는 것이 혹시 어떤 말씀이신가요? 바로 위 소헌님께서 고쳐주신 코드를 보면 변동사항이 없는 것 같아 보이는데.. 이해가 잘 되지 않습니다! 🥲

Reply:
아하 ㅎㅎ 간격이 다릅니다 지현님 :) 코드 앞에 여백이요! :)

Reply:
아! 그 말이시군요. 수정하겠습니다.